coverage.py

```diff
@@ -421,27 +354,6 @@ def process_work(self, identifier):
             error_msg = "500; " + "Work could not be calculated for %r" % identifier
             raise RuntimeError(error_msg)
 
-    def resolve_equivalent_oclc_identifiers(self, identifier):
```

Comment:
No longer required. It turns out `LinkedDataCoverageProvider` was already doing all of this!